build/azDevOps/azure/azure-pipelines-netcore-k8s.yml

```diff
@@ -25,9 +25,7 @@ resources:
     - repository: templates
       type: github
       name: amido/stacks-pipeline-templates
-      ref: refs/tags/v2.0.4
-      # EXCHANGE THIS FOR YOUR OWN ENDPOINT CONNECTION TO GITHUB
-      # REPOSITORY IS PUBLIC
+      ref: refs/tags/v2.0.1
```

Comment:
Is there a reason you've rolled back in the Pipeline Templates?

Reply:
No that is a mistake which I will correct